mods/gennext/README.md

```diff
@@ -236,6 +237,14 @@ New mechanic: Intrinsics:
 
   - Mismagius: Cursed Body
 
+  - Rotom:
+    - Normal: Motor Drive
+    - Heat: Flash Fire
+    - Wash: Hydration
+    - Mow: Sap Sipper
+    - Fan: Gale Wings
```

Comment:
Out of boredom, I have compiled a list of abilities I think would fit on Rotom-S:
- **Infiltrator:** the wind produced from a fan would logically go around any given object to a certain degree, hence Infiltrator vs Substitutes
- **Speed Boost:** as a fan is turned on, it rapidly begins to spin faster and faster (and can spin EVEN FASTER if the setting on it is changed!)
- **Synchronize:** If a Pokemon uses something like Will-O-Wisp on Rotom-S, it makes sense that some of the wisps would be blown back towards the user